components/Navbar.jsx

```diff
@@ -0,0 +1,3 @@
+function Navbar() {
+  return (<div>NavBar</div>);
```

Comment:
Codacy has a fix for the issue: [Replace `(<div>NavBar</div>)` with `<div>NavBar</div>`](https://app.codacy.com/gh/thamizhMsys/testRepo6/pullRequest?prid=9650719)
```suggestion
  return <div>NavBar</div>;
```